src/seedu/addressbook/AddressBook.java

```diff
@@ -306,13 +311,49 @@ private static void setupDefaultFileForStorage() {
 
     /**
      * Returns true if the given file is acceptable.
-     * The file path is acceptable if it ends in '.txt'
+     * The file path is acceptable if it does not contain any reserved string,
```

Comment:
What's a 'reserved string'? Is there a better way to explain the behavior of this method?